pkg/reconciler/common/stages_test.go

```diff
@@ -30,17 +30,184 @@ import (
 )
 
 func TestStagesExecute(t *testing.T) {
-	os.Setenv(KoEnvKey, "testdata/kodata")
+	koPath := "testdata/kodata"
+	os.Setenv(KoEnvKey, koPath)
 	defer os.Unsetenv(KoEnvKey)
-	manifest, _ := mf.ManifestFrom(mf.Slice{})
-	stages := Stages{AppendTarget, AppendInstalled}
-	util.AssertEqual(t, len(manifest.Resources()), 0)
-	stages.Execute(context.TODO(), &manifest, &v1alpha1.KnativeServing{Spec: v1alpha1.KnativeServingSpec{
-		CommonSpec: v1alpha1.CommonSpec{
-			Version: "0.16.0",
+
+	tests := []struct {
+		name                   string
+		component              v1alpha1.KComponent
+		expectedManifestsPath  string
+		expectedContainingPath string
```

Comment:
`expectedContainingPath` is not used in the test.

Reply:
removed.